### ISSUE TITLE
Renaming tag deletes the container?

### ISSUE THREAD

**Issue by kaihendry** [containers/skopeo#1981]
I'm trying to rename a $CI_COMMIT_SHORT_SHA tagged image to foo. However when I try delete the old tag $CI_COMMIT_SHORT_SHA, the foo image is also removed. Is this expected behaviour?
```

   ❯ skopeo --version
    skopeo version 1.12.0
    ❯ skopeo list-tags docker://registry.gitlab.com/kaihendry/boda
    {
        "Repository": "registry.gitlab.com/kaihendry/boda",
        "Tags": [
            "e5c33170"
        ]
    }
    
    ❯ skopeo copy docker://registry.gitlab.com/kaihendry/boda:e5c33170 docker://registry.gitlab.com/kaihendry/boda:foo
    Getting image source signatures
    Copying blob aeea67982490 skipped: already exists
    Copying config f1aaf65f15 done
    Writing manifest to image destination
    Storing signatures
    
    skopeo list-tags docker://registry.gitlab.com/kaihendry/boda
    {
        "Repository": "registry.gitlab.com/kaihendry/boda",
        "Tags": [
            "e5c33170",
            "foo"
        ]
    }
    ❯ skopeo delete docker://registry.gitlab.com/kaihendry/boda:e5c33170
    ❯ skopeo list-tags docker://registry.gitlab.com/kaihendry/boda
    {
        "Repository": "registry.gitlab.com/kaihendry/boda",
        "Tags": []
    }
```

**Comment by mtrmac** [comment]
Thanks for your report.

Yes, this is currently the way deletion is implemented: #1432 .

(Some?) registries now do support deleting tags, but c/image and Skopeo doe not implement that yet.

**Comment by kaihendry** [comment]
Ok, thanks. I did notice btw that crane 0.14.0-1 deletes a tag as expected.
